### DISCUSSION TITLE
Loss Convergence

### DISCUSSION THREAD

**Discussion by alib-l** [deepmodeling/deepmd-kit#479] [Q&A]
Hello, 
I am training a model using the "se_a" type. I am using "ReLu" as an activation function, and the "batch size" is 1. The problem is that the loss converges slowly and there is a significant difference between test and train loss (after 170,000 batches, the training loss is: 12, and the test loss is:13). Also, the force losses are increasing. What is your advice for better convergence?
I attached the "lcurve.out" plot for total and force losses. I would be thankful if you help me with this issue.
![Force](https://user-images.githubusercontent.com/68351594/113709908-52320e00-96f8-11eb-90bd-40d0e3239818.png)
![Total](https://user-images.githubusercontent.com/68351594/113709917-552cfe80-96f8-11eb-8609-e13e43584749.png)

**Comment by njzjz** [comment]
I don't recommend to use ReLU, since ReLU function is not derivable at x=0, and its derivative is not continuous at x<0 and x>0. However, the force is the negative energy gradient.

![ReLU](https://upload.wikimedia.org/wikipedia/commons/c/c9/Ramp_function.svg)

**Reply by alib-l** [reply]
I also checked the "exp", but there was no change in the losses.

**Reply by njzjz** [reply]
From your plot, I am assuming your training data is not correct. Do you use the correct unit as shown in https://github.com/deepmodeling/deepmd-kit/blob/v1.3.3/doc/use-deepmd-kit.md#prepare-data?

**Reply by njzjz** [reply]
I am assuming your energy are inconsistent with forces. You may check it by only using energy or forces and see the loss again.

**Reply by njzjz** [reply]
I believe your forces are not correct. Can you provide more information, for example, how you take the data?

**Reply by alib-l** [reply]
I provided all of the data from the OUTCAR using my own python script, and using the below tags. 
"TOTAL-FORCE" for "force.raw"
"POSITIONS" for "coord.raw"
"in KB" for "virial.raw"
"free energy TOTEN" for "energy.raw"

Also, I checked my raw files with "dpdata".

**Reply by WU2JING** [reply]
Hello, i also meet the problem.Have you solved the problem？

**Reply by alib-l** [reply]
@WJforu Not yet, HBU?
